chromium/options.js

```diff
@@ -0,0 +1,28 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+  let json_data;
+  let import_button = document.querySelector("#import");
+
+  function import_json(e) {
+    e.preventDefault();
+
+    let settings = JSON.parse(json_data);
+    sendMessage("import_settings", settings, resp => {
+      document.querySelector("#import-confirmed").style.display = "block";
+      document.querySelector("form").style.display = "none";
```

Comment:
Maybe find a `form` by `id` instead of simply taking the first one in the document?